cocotb/xunit_reporter.py

```diff
@@ -28,41 +28,9 @@
 from xml.etree.ElementTree import Element, SubElement
 import xml.etree.ElementTree as ET
 
-import mmap
-from io import StringIO
-
 TRUNCATE_LINES = 100
 
 
-# file from  http://stackoverflow.com/questions/136168/get-last-n-lines-of-a-file-with-python-similar-to-tail
-class File(StringIO):
```

Comment:
Probably should have a release note, "The undocumented `cocotb.xunit_report.File` class has been removed"